sparta/sparta/events/PhasedPayloadEvent.hpp

```diff
@@ -175,7 +176,9 @@ namespace sparta
         //! reusable.
         void reclaimProxy_(typename ProxyInflightList::iterator & pl_location) {
             sparta_assert(pl_location != inflight_pl_.end());
-            free_pl_[free_idx_++] = *pl_location;
+            if constexpr(MetaStruct::is_any_pointer<DataT>::value) {
+                (*pl_location)->setPayload_(nullptr);
+            }            free_pl_[free_idx_++] = *pl_location;
```

Comment:
Is `}            free_pl_[free_idx_++] = *pl_location;` two code lines that accidentally got merged into one code line? Or is this intended to be one line of code?

Reply:
Copy/paste error.  Thanks for finding.